doc/api/v8.md

```diff
@@ -134,3 +134,157 @@ setTimeout(function() { v8.setFlagsFromString('--notrace_gc'); }, 60e3);
 [`GetHeapSpaceStatistics`]: https://v8docs.nodesource.com/node-5.0/d5/dda/classv8_1_1_isolate.html#ac673576f24fdc7a33378f8f57e1d13a4
+
+## Serialization API
+
+> Stability: 1 - Experimental
+
+### class: v8.Serializer
+<!--
+added: REPLACEME
+-->
+
+#### new Serializer([options])
+
+* `options` {Object} Set of configurable options to set on the serializer.
+  Can have the following fields:
+  * `getDataCloneError` {Function} A callback that takes a single `message`
+    argument and returns an error object. Defaults to the `Error` constructor.
+  * `writeHostObject` {Function} A callback that takes a single `object`
+    argument and is used to write some kind of host object, if possible. If not,
+    a suitable exception should be thrown.
+
+Creates a new `Serializer` object.
+
+#### serializer.writeHeader()
+
+Writes out a header, which includes the serialization format version.
+
+#### serializer.writeValue(value)
+
+* Returns: {Boolean}
+
+Serializes a JavaScript value and adds the serialized representation to the
+internal buffer.
+
+#### serializer.releaseBuffer()
+
+Returns the stored internal buffer. This serializer should not be used once
+the buffer is released. Calling this method results in undefined behavior
+if a previous write has failed.
+
+#### serializer.transferArrayBuffer(id, arrayBuffer)
+
+* `id` {Integer} A 32-bit unsigned integer.
+* `arrayBuffer` {ArrayBuffer|SharedArrayBuffer} An `ArrayBuffer` instance.
+
+Marks an `ArrayBuffer` as havings its contents transferred out of band.
+Pass the corresponding `ArrayBuffer` in the deserializing context to
+[`deserializer.transferArrayBuffer()`][].
+
+#### serializer.writeUint32(value)
+
+* `value` {Integer}
+
+Write a raw 32-bit unsigned integer.
+For use inside of a custom `options.writeHostObject`.
+
+#### serializer.writeUint64(hi, lo)
+
+* `hi` {Integer}
+* `lo` {Integer}
+
+Write a raw 64-bit unsigned integer, split into high and low 32-bit parts.
+For use inside of a custom `options.writeHostObject`.
+
+#### serializer.writeDouble(value)
+
+* `value` {Number}
+
+Write a JS `number` value.
+For use inside of a custom `options.writeHostObject`.
+
+#### serializer.writeRawBytes(buffer)
+
+* `buffer` {Buffer|Uint8Array}
+
+Write raw bytes into the serializer’s internal buffer. The deserializer
+will require a way to compute the length of the buffer.
+For use inside of a custom `options.writeHostObject`.
+
+### class: v8.Deserializer
+<!--
+added: REPLACEME
+-->
+
+#### new Deserializer(buffer, [options])
+
+* `buffer` {Buffer|Uint8Array} A buffer returned by [`serializer.releaseBuffer()`][].
+* `options` {Object} Set of configurable options to set on the serializer.
+  Can have the following fields:
+  * `readHostObject` {Function} A callback that takes no arguments and is
+    used to host object, if possible. If not, a suitable exception should
+    be thrown.
+
+Creates a new `Deserializer` object.
+
+#### deserializer.readHeader()
+
+* Returns: {Boolean}
```

Comment:
> true/false means?

I’ve removed the return type from the documentation. Right now it’s always `true` except when an error is thrown.